src/rcms/fm/app/level1/HCALStateNotificationHandler.java

```diff
@@ -216,8 +213,6 @@ else if ( notification.getToState().equals(HCALStates.HALTED.toString()) ) {
       if(taskSequence == null) {
 
         setTimeoutThread(false);
-        String infomsg = "Received a State Notification while taskSequence is null \n";
-
         logger.debug("FM is in local mode");
```

Comment:
Makes more sense to have 
`logger.debug(infomsg);`

Reply:
Do you mean to switch line 221/216 to that? Or to add the a line `debug(infomsg)` below? In any case, Debug-level messages get discarded at P5 so I'm not sure why we ever want to keep debug-level messages.

Reply:
yes, i'd replace 221/216 with `logger.debug(infomsg);`
The point of having debug messages is to leave some stepping stones for debugging recurring issues, like state-transition notification, so that we can just move the log to warn/info level and get the message we want.

Reply:
ok. did that in 3d7ecd3